### PR TITLE
Create preprocess.md

### PR DESCRIPTION
**Pull request by FaezeK**

### CODE REVIEW

**Comment by creisle** [comment]
can you add a link to this from the main README?

**Comment by FaezeK** [comment]
> can you add a link to this from the main README?

Do you mean the Reports directory README? the main/root README doesn't have a link to any of the other reports

**Comment by creisle** [comment]
> Do you mean the Reports directory README? the main/root README doesn't have a link to any of the other reports

yup sorry, too many READMEs, i means the main one in the reports dir

**Comment by FaezeK** [comment]
> > Do you mean the Reports directory README? the main/root README doesn't have a link to any of the other reports
> 
> yup sorry, too many READMEs, i means the main one in the reports dir

Reports are added to README

File: reports/preprocess.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
+preprocess
+================
+Faeze Keshavarz
+09/04/2021
+
+**Inputs:** 1 - metadata file obtained by extracting required columns
```
Comment (creisle) [inline reports/preprocess.md:6]: super minor, but if you're going to number them anyway, why not do a md numbered list?

```md
**Inputs:**

1. metadata file obtained by extracting required columns
2. original Hiseq data that contains both ra...
```

**Review by creisle — DISMISSED** [review]

**Review by creisle — DISMISSED** [review]

File: reports/preprocess.md (hunk 1 above)
Comment (FaezeK) [reply reports/preprocess.md]: While I was at it, I also fixed the numbers in DE_btw_subtypes report

**Review by clmorrison — APPROVED** [review]

**Pull request merged**